test/amqp-integration-test.py

```diff
@@ -222,14 +223,32 @@ def cleanup():
     else:
         print("\n\nFAILURE %d" % exit_status)
 
-
 exit_status = ExitStatus.OK
 tempdir = tempfile.mkdtemp()
+
+parser = argparse.ArgumentParser(description='Run integration tests')
+parser.add_argument('--no-node', dest="run_node", action="store_false",
```

Comment:
Why not have a --tests= flag, which can contain "node", "python", or "node,python", and defaults to "node,python" if it's absent? That gives us a little more flexibility to add new test types to the integration test in the future.